src/lib/components/FormModel/children/Group/group.test.js

```diff
@@ -0,0 +1,8 @@
+import 'jest-dom/extend-expect';
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+test('', () => {
+  expect(true).toBeTruthy();
```

Comment:
This test does not do anything. Shouldn't it instantiate a `Group` component and test its rendering ?

Reply:
Yes, good catch. I think this and Multiple aren't included normally because they aren't discrete things, they require external dependencies. Meaning a "Group" is nonsensical by itself.  It needs to render subcomponents, so this will require a bit of planning for the best kind of test.